part-2-server/server.js

```diff
@@ -0,0 +1,55 @@
+
+var http = require('http'); 
+var fs = require('fs'); 
+
+var server = http.createServer(function(request, response){
+
+var url = request.url;
+
+
+  //get url & create if/else  for differernt urls
+  if (url === '/'){ 
+    fs.readFile('index.html', function(err, data) {
+            response.write(data.toString());
+            response.end();
+      });
+  }
+
+
+  else if(url === '/page-2' || url === '/page-3') {
+    fs.readFile('page-2.html', function(err, data) {
+            response.write(data.toString());
+            response.end();
+        });
+  }
+
+
+
+
+  else if(url === '/poem-') {
```

Comment:
Need to get a substring of the URL, otherwise this conditional will never be triggered